docs/commands/join.md

```diff
@@ -42,53 +40,55 @@ redocly join --version
-| --prefix-tags-with-filename        | string   | Prefix tags with property value from file name. See the [prefix-tags-with-filename section](#prefix-tags-with-filename) below.                                                                             |
-| --prefix-tags-with-info-prop       | boolean  | Prefix tags with property value from info object. See the [prefix-tags-with-info-prop](#prefix-tags-with-info-prop) section.                                                                               |
+| --output, -o                       | string   | Name for the joined output file. Defaults to `openapi.yaml` or `openapi.json` (depends on the extension of the first input file). **If the file already exists, it's overwritten.**                        |
+| --prefix-components-with-info-prop | string   | Prefix components with property value from info object. See the [prefix-components-with-info-prop](#use-prefix-components-with-info-prop-option) section.                                                  |
+| --prefix-tags-with-filename        | boolean  | Prefix tags with property value from file name. See the [prefix-tags-with-filename](#use-prefix-tags-with-filename-option) section.                                                                        |
+| --prefix-tags-with-info-prop       | string   | Prefix tags with property value from info object. See the [prefix-tags-with-info-prop](#use-prefix-tags-with-info-prop-option) section.                                                                    |
 | --version                          | boolean  | Show version number.                                                                                                                                                                                       |
-| --without-x-tag-groups             | boolean  | Skip automated `x-tagGroups` creation. See the [without-x-tag-groups](#without-x-tag-groups) section.                                                                                                      |
+| --without-x-tag-groups             | boolean  | Skip automated `x-tagGroups` creation. See the [without-x-tag-groups](#use-without-x-tag-groups-option) section.                                                                                           |
+
+{% admonition type="warning" %}
+These options are mutually exclusive: `without-x-tag-groups`, `prefix-tags-with-filename`, and `prefix-tags-with-info-prop`.
+{% /admonition %}
 
 ## Examples
 
-### Array of paths
+### Specify array of paths
 
-**Command**
+This command combines the API description files that are passed in:
 
 ```bash
 redocly join first-api.yaml second-api.json
 ```
 
-**Output**
+The output file `openapi.yaml` is created in the working directory:
 
-```bash
+<pre>
```

Comment:
Why change from markdown syntax to html?

Reply:
@adamaltman  this is the purpose of the issue. We had a lot of either commands in text, or a wall of repeated code samples. The brief is to differentiate between the user's input, and displayed output examples.